scanner/config/config.go

```diff
@@ -18,13 +19,20 @@ var (
 		HTTPListenAddr: "127.0.0.1:9443",
 		GRPCListenAddr: "127.0.0.1:8443",
 		Indexer: IndexerConfig{
-			Enable:          true,
-			DBConnString:    "postgresql:///postgres?host=/var/run/postgresql",
+			Enable: true,
+			Database: Database{
+				ConnString:   "host=/var/run/postgresql",
```

Comment:
This could be empty, and it would default to the same path. Nonetheless, requiring something makes things clearer and less prone to errors.